app/components/widgets/LoadingPaperWallet.js

```diff
@@ -0,0 +1,12 @@
+// @flow
```

Comment:
I don't think we gonna reuse this, it's better to directly add to respective component.

Reply:
Okay, will fix